Add moutoussis2018

kozzy97 opened the pull request:
Add dataset from Moutoussis et al. (2018).

The dataset contains single prompts for each participant, including any follow-ups they took part in.

Notes:
* Gender, age, IQ, and other questionnaire scores cannot be discerned from available datasets (columns are unlabelled).
* There are potentially some extra and/or missing responses from the 18-month follow ups. Some participant IDs appear in the 18-month follow-up that don't appear in the baseline. The authors also report removing some participants due to low-quality responses. It's not clear how to eliminate them from the datasets alone.

marcelbinz commented:
looking good. A few minor things before merging: 

* sometimes it says "You chose <<3>>." which seems inconsistent with the instructions. What is going here?
* is it possible to add something like: "You come back to the lab X month later." In between the sessions?
* please upload a zipped version of the jsonl instead (to save storage and make data leakage harder).

kozzy97 commented:
Hi @marcelbinz 

I have made the changes:
1. I really don't know what is going on with the <<3>> cases, there is next to no supporting documentation with the dataset. So I've removed any trials with 3s in them, and also removed any trial indexing. Now it just says `In the next trial...`
2. I have added `You have returned to the lab X months later`, with `X={6, 12, 18}` depending on the participant.
3. I have zipped the jsonl

Let me know if there are any issues with it!

marcelbinz commented:
Thanks again for the contribution!
